epub34/authoring/index.html

```diff
@@ -5795,27 +5807,38 @@ <h4>NCX</h4>
 			<h2>EPUB content documents</h2>
 
 			<section id="sec-xhtml">
-				<h3>XHTML content documents</h3>
+				<h3>HTML content documents</h3>
+
+				<div class="ednote">
+					<p>Support for the HTML syntax is still an open issue in the EPUB 3.4 revision. This specification
+						introduces it as a core media type and also minimally adds requirements for its authoring in
+						this section to begin making readers aware of the proposed change.</p>
```

Comment:
```suggestion
					<p>Support for the HTML syntax is still an open issue in the EPUB 3.4 revision. It is <b><i>at risk</i></b>, depending on authors' and implementers' feedbacks . This specification
						introduces it as a core media type and also minimally adds requirements for its authoring in
						this section to begin making readers aware of the proposed change.</p>
```

Declaring a feature officially "at risk" means that we can enter CR with this feature with the possibility to pull it at the last minute if implementations fail. I realize it is very early to talk about CR, but it may be worth emphasizing this as soon as possible.